noisereduction/Utils.h

```diff
@@ -3,11 +3,14 @@
 #include <sndfile.h>
 #include <assert.h>
 
+#define WINDOWS (defined(WIN32) || defined(_WIN32) || defined(__WIN32))
+
 struct SndContext {
     SNDFILE *file;
     SF_INFO info;
 };
 
+#ifndef WINDOWS
 class SndMmap {
```

Comment:
maybe better to put it in a separate file?

Reply:
Yeah, I agree, I will do that.

Reply:
yeah makes sense